### PR TITLE
Conductor UI Release - 3.3.0

### PR DESCRIPTION
**Pull request by peterlau**
Pull Request type
----

- [ ] Bugfix
- [X] Feature
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] Other (please describe):

Changes in this PR
The Conductor 3 UI is now available on the 3.3.0 branch.

What's New?
* Improved visual hierarchy to accommodate both existing and future user stories in observability and workflow creation.
* New component design system based on Material UI and React Hooks
* Enhanced workflow graph visualization fixing edge-cases.
* Scalable handling of high-cardinality dynamic forks and retries
* Performance improvements and caching

### CODE REVIEW

**Comment by jasondavindev** [comment]
Do you have prints of the changes?

File: README.md [hunk 1]
```diff
@@ -2,45 +2,58 @@
...
 
-## Running Conductor
-The latest version is [![Github release](https://img.shields.io/github/v/release/Netflix/conductor.svg)](https://GitHub.com/Netflix/conductor/releases). Conductor can be started up with an in-memory persistence and queue implementation. It is not recommended for production use but can come in handy for quickly evaluating what Conductor's all about. For actual production use-cases, please use one of the supported persistence and queue implementations.
+## Getting Started - Building & Running Conductor
+### Docker
+The easiest way to get started is with Docker containers. Please follow the instructions [here](https://github.com/Netflix/conductor/tree/main/docker). The server and UI can also be built from source separately.
 
-If Docker is installed, please follow the instructions [here](https://github.com/Netflix/conductor/tree/main/docker). 
+### Conductor Server From Source
+Conductor Server is a [Spring Boot](https://spring.io/projects/spring-boot) project and follows all applicable conventions. First, ensure that Java JDK 14 is installed.
```
Comment (aravindanr) [inline README.md:26]: Conductor works with Java 11 and above. `14` is not a requirement.

File: README.md [hunk 2]
```diff
@@ -2,45 +2,58 @@
...
 
-If Java 11 or above is installed, Conductor [publishes](https://mvnrepository.com/artifact/com.netflix.conductor/conductor-server) an executable jar that can be run using,
+#### Development
+The server can be started locally by running `./gradlew bootRun` from the project root. This will start up Conductor with an in-memory persistence and queue implementation. It is not recommended for production use but can come in handy for quickly evaluating what Conductor's all about. For actual production use-cases, please use one of the supported persistence and queue implementations.
 
+You can verify the development server is up by navigating to `http://localhost:8080/swagger-ui/index.html` in a browser.
+
+#### Production Build
+Running `./gradlew build -x test` from the project root builds the project into the `/build` directory.
```
Comment (aravindanr) [inline README.md:34]: `./gradlew assemble` will do the same.

File: docker/server/config/config-local.properties [hunk 3]
```diff
@@ -30,3 +30,5 @@ conductor.indexing.enabled=false
 
 # Load sample kitchen sink workflow
 loadSample=true
+
+conductor.elasticsearch.clusteHealthColor=yellow
```
Comment (aravindanr) [inline docker/server/config/config-local.properties:34]: typo, `clusterHealthColor`. Why is it set as `yellow`?

**Review by aravindanr — APPROVED** [review]

**Pull request merged**